content/documents/child-products.mdx

```diff
@@ -0,0 +1,90 @@
+---
+type:           article
+title:          Child Products
+description:    Introducing Child Products, a new feature set that enables developers to define a product as a Parent and subsequently create Child products.
+author:         gvo
+publishDate:    2023-02-09
+updatedDate:    2023-02-09
+tags:           ["Child Products", "Products", "Parent", "Child", "Inheritance"]
+---
+
+## Benefits
+
+- Allows creating product hierarchies without having to generate Variants, which may not be ideal for every marketplace.
+- Allows defining custom pricing per Child product without having to use Spec Markups, which may be cumbersome to set up, and don’t have Multi-Currency support at this time.
+- Child Products can be returned in product lists, which is not possible with Variants.
+
+## New Properties
+
+### Product.IsParent
+- Used to define if a product is a Parent. 
+-- If true, Product.ParentID must be null as Parent products may not be Child products.
```

Comment:
I believe this should just be `   -` instead of `--`

![image](https://user-images.githubusercontent.com/10764202/217880133-d6699e0f-906d-4ee2-84f5-51927aa1020a.png)